File: YMCAUI/CommonClasses/LoanStatus.vb
'************************************************************************************************************************/
' Author:                   : Manthan Rajguru
' Created on:               : 04/11/2018
' Summary of Functionality  : Provides Loan Statuses (WEB/YRS)
' Declared in Version       : 20.6.0 | YRS-AT-4017 -  YRS enh: Loan EFT Project: "new" Loan Admin webpages (approve/decline/process) 
'
'************************************************************************************************************************/
' REVISION HISTORY:
' ------------------------------------------------------------------------------------------------------
' Developer Name              | Date         | Version No      | Ticket
' ------------------------------------------------------------------------------------------------------
' 			                  | 	         |		           | 
' ------------------------------------------------------------------------------------------------------
'************************************************************************************************************************/
''' <summary>
''' Structure type for loan statuses which can be used in loan modules for checking loan status instead of hardcoded
''' </summary>
''' <remarks></remarks>
Public Structure LoanStatus
    Public Shared ACCEPTED As String = "ACCEPTED"
    Public Shared APPROVED As String = "APPROVED"
    Public Shared CANCEL As String = "CANCEL"
    Public Shared DECLINED As String = "DECLINED"
    Public Shared EXP As String = "EXP"
    Public Shared PAID As String = "PAID"
    Public Shared PEND As String = "PEND"
    Public Shared REJECTED As String = "REJECTED"
    Public Shared WITHDRAWN As String = "WITHDRAWN"
End Structure

File: YMCAObjects/LoanStatus.cs
/************************************************************************************************************************/
// Author: Sanjay GS Rawat
// Created on: 10/10/2018
// Summary of Functionality: Loan status structure
// Declared in Version: 20.6.0 | YRS-AT-3101 -  YRS enh: EFT: Loans ( FIRST EFT PROJECT) - updates for "Payment Manager" (TrackIT 33024) 
//
/************************************************************************************************************************/
// REVISION HISTORY:
// ------------------------------------------------------------------------------------------------------
// Developer Name                   | Date          | Version No        | Ticket
// ------------------------------------------------------------------------------------------------------
// Megha Lad	                    | 2019.01.17   |	20.6.2           | YRS-AT-3157 - YRS enh-audit trail db table to track Loan freeze and unfreeze activity and copy emails to IDM (TrackIT 27438)
// ------------------------------------------------------------------------------------------------------
//************************************************************************************************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace YMCAObjects
{
    public struct LoanStatus
    {
        public static string ACCEPTED = "ACCEPTED";
        public static string APPROVED = "APPROVED";
        public static string CANCEL = "CANCEL";
        public static string DECLINED = "DECLINED";
        public static string EXP = "EXP";
        public static string PAID = "PAID";
        public static string PEND = "PEND";
        public static string REJECTED = "REJECTED";
        public static string WITHDRAWN = "WITHDRAWN";
        public static string DISB = "DISB";
        public static string PROOF = "PROOF";
        //START : ML | 2019.01.17 | YRS-AT-3157 | Loan status for Freeze unfreeze to use in SendMail call
        public static string FREEZE = "FREEZE";
        public static string UNFREEZE = "UNFREEZE";
        //END : ML | 2019.01.17 | YRS-AT-3157 | Loan status for Freeze unfreeze to use in SendMail call        
    }
}
